Assets/Scripts/InventoryScripts/ManagerScript.cs

```diff
@@ -5,80 +5,59 @@
-                invPanel.SetActive(false);      //sets the GameObject as unactive
-                journalInvPanel.SetActive(false);      //sets the GameObject as unactive
-                itemInfo.SetActive(false);
+                _invPanel.SetActive(false);      //sets the GameObject as unactive
+                _journalInvPanel.SetActive(false);      //sets the GameObject as unactive
+                _itemInfo.SetActive(false);
             }
         }
     }
 
     public void LoadInfo(int _buttonNumber)         //function that Loads information onto the Description panel
     {
-        nameOfItem.text = inventoryScript.items[_buttonNumber].GetComponent<ItemInfo>().itemName;                   //Loads name of item that was pressed onto Description panel
-        descriptionOfItem.text = inventoryScript.items[_buttonNumber].GetComponent<ItemInfo>().itemDescription;     //Loads description of item that was pressed onto Description panel
+        _nameOfItem.text = _inventoryScript._items[_buttonNumber].GetComponent<ItemInfo>().itemName;                   //Loads name of item that was pressed onto Description panel
+        _descriptionOfItem.text = _inventoryScript._items[_buttonNumber].GetComponent<ItemInfo>().itemDescription;     //Loads description of item that was pressed onto Description panel
     }
 
-    public void LoadGood()
+    public void LoadPageRiddles(ItemInfo item)
     {
-        nameOfItem.text = "Good Journal";
-        descriptionOfItem.text = "";
-        foreach (GameObject page in journalScript.GoodPages)
+        _descriptionOfItem.text = "";
+        _nameOfItem.text = item.itemName;
+        foreach (var PAGE in _journalScript._books)
         {
-            if(page != null) descriptionOfItem.text += page.GetComponent<ItemInfo>().itemDescription + System.Environment.NewLine;
-        }
-    }
-    public void LoadEvil()
-    {
-        nameOfItem.text = "Evil Journal";
-        descriptionOfItem.text = "";
-        foreach (GameObject page in journalScript.EvilPages)
-        {
-            if (page != null) descriptionOfItem.text += page.GetComponent<ItemInfo>().itemDescription + System.Environment.NewLine;
-        }
-    }
-    public void LoadDeceitful()
-    {
-        nameOfItem.text = "Deceitful Journal";
-        descriptionOfItem.text = "";
-        foreach (GameObject page in journalScript.DeceitfulPages)
-        {
-            if (page != null) descriptionOfItem.text += page.GetComponent<ItemInfo>().itemDescription + System.Environment.NewLine;
-        }
-    }
-    public void LoadUnfortunate()
-    {
-        nameOfItem.text = "Unfortunate Journal";
-        descriptionOfItem.text = "";
-        foreach (GameObject page in journalScript.UnfortunatePages)
-        {
-            if (page != null) descriptionOfItem.text += page.GetComponent<ItemInfo>().itemDescription + System.Environment.NewLine;
+            if (PAGE == null) continue;
```

Comment:
I don't think this should be `SCREAMING_CASE`. instead its name should probably be `page`.